README.md

```diff
@@ -151,6 +151,35 @@ class ChangePassword {
 });
 ```
 
+### VueJs
+```js
+import Vue from 'vue';
+import Logger from 'utils/logger';
+import Notification from 'utils/notification';
+
+Vue.component('change-password', {
+  template: '<div>{{ /* template */ }}</div>'
+  data() {
+      return {
+        password: ''
+      };
+  },
+  methods: {
+      changePassword() {
+          this.$store.dispatch(
```

Comment:
Is it some kind of a Redux usage? I wanted to keep this example as simple as possible. Can we use `Auth` service directly as in the React example?

Reply:
Yes, it's like Redux for Vue (Vuex). I thought that the React example was with a store because it was under an action. Happy to simplify it to be the same as the React example.

Reply:
Great, thanks. I'll create a separate `Redux` section later on.